app/main.py

```
@@ -0,0 +1,55 @@
+
+
+# Startup event to create database tables
+@app.on_event("startup")
+async def on_startup() -> None:
+    async with engine.begin() as conn:
+        await conn.run_sync(models.Base.metadata.create_all)
+
+
+# Get all ToDo items
+@app.get("/todos")
+async def read_todos(db: AsyncSession = Depends(get_db)) -> list[schemas.ToDoResponse]:
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)